composer.json

```diff
@@ -43,7 +43,9 @@
         "doctrine/dbal": "~2.2",
         "swiftmailer/swiftmailer": "~5",
         "monolog/monolog": "^1.4.1",
-        "phpunit/phpunit": "^5 || ^6 || ^7 || ^8"
+        "phpunit/phpunit": "^5 || ^6 || ^7 || ^8",
```

Comment:
Use only 8 now, since the old ones would be a lie (we use new functionality now).,